scripts/bootstrap-ansible.sh

```diff
@@ -138,6 +138,7 @@ EOF
 
 chmod +x /usr/local/bin/ceph-ansible
 echo "ceph-ansible wrapper created."
+ANSIBLE_BINARY=/opt/rpc-ceph_ansible-runtime/bin/ansible
```

Comment:
Remove the lines added here (and below) and add:
export ANSIBLE_BINARY="${ANSIBLE_BINARY:-ceph-ansible}"

to the top of the file, with the other exports. Should be good to go after that. It's just an ease of use binary that points straight to the venv, saves you having to manually reference the venv.

Reply:
yep, that's a better fix